test/fixtures/output/schema-registry/schema/describe-refs-id.golden

```diff
@@ -15,10 +16,14 @@
         }
       ],
       "schema": "schema0",
-      "schemaType": "AVRO"
+      "metadata": {},
+      "ruleset": {}
```

Comment:
Looks like these are new fields that didn't exist before. Can we mark them as `omitempty` maybe?